docs/getting-started/installation.md

```diff
@@ -26,48 +26,32 @@ Prefect requires Python 3.8 or newer.
 
 We recommend installing Prefect using a Python virtual environment manager such as `pipenv`, `conda`, or `virtualenv`/`venv`.
 
-!!! success "Upgrading from Prefect 1 to Prefect 2"
-    If you're upgrading from Prefect 1 to Prefect 2, we recommend creating a new environment. Should you encounter any issues when upgrading, this ensures being able to roll back to a known working state easily.
-
-!!! warning "Windows and Linux requirements"
+!!! info "Windows and Linux requirements"
     See [Windows installation notes](#windows-installation-notes) and [Linux installation notes](#linux-installation-notes) for details on additional installation requirements and considerations.
 
 ## Install Prefect
 
 The following sections describe how to install Prefect in your development or execution environment.
 
-### Installing the latest version
+### Installing the most recent version
 
-Prefect is published as a Python package. To install the latest Prefect release, run the following in a shell or terminal session:
+Prefect is published as a Python package. To install the most recent release or upgrade an existing Prefect install, run the following command in your terminal:
 
 <div class="terminal">
 ```bash
 pip install -U prefect
 ```
 </div>
 
-To install a specific version, specify the version, such as:
+To install a specific version, specify the version number like this:
 
 <div class="terminal">
 ```bash
 pip install -U "prefect==2.10.4"
 ```
 </div>
 
-Find the available release versions in the [Prefect Release Notes](https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md) or the [PyPI release history](https://pypi.org/project/prefect/#history).
-
-### Installing the bleeding edge
-
-If you'd like to test with the most up-to-date code, you can install directly off the `main` branch on GitHub:
-
-<div class="terminal">
-```bash
-pip install -U git+https://github.com/PrefectHQ/prefect
-```
-</div>
-
-!!! warning "The `main` branch may not be stable"
-    Please be aware that this method installs unreleased code and may not be stable.
+See available release versions in the [Prefect Release Notes](https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md).
```

Comment:
Not strongly held, but I think Brian and I were both thinking it could cut down on the content on the page. Further I was thinking that most anyone who needs to install bleeding edge version probably knows how to do so. I can leave it in.